### PR TITLE
chore: updates repo name to uds-package-dubbd

### PR DESCRIPTION
**Pull request by UncleGedd**
Updates repo name to uds-package-dubbd

### CODE REVIEW

File: .github/workflows/pipeline.yaml [hunk 1]
```diff
@@ -21,7 +21,6 @@ jobs:
       - name: Install Zarf
         uses: defenseunicorns/setup-zarf@main
         with:
-          # renovate: datasource=github-tags depName=defenseunicorns/zarf versioning=semver
```
[the remark this answers is not in the document]
Comment (UncleGedd) [reply .github/workflows/pipeline.yaml:24]: Not sure...I saw it was commented out so I removed it along with the other comments in that file. I think @bdfinst is also refactoring this?

**Review by cmharden — DISMISSED** [review]

File: .github/workflows/pipeline.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (UncleGedd) [reply .github/workflows/pipeline.yaml:24]: TIL! Will add it back

**Review by MxNxPx — COMMENTED** [review]
as discussed - the zarf package names are not being changed with this PR (may change later)...
https://github.com/defenseunicorns/zarf-package-big-bang/blob/update-repo-name/defense-unicorns-distro/zarf.yaml#L3
https://github.com/defenseunicorns/zarf-package-big-bang/blob/update-repo-name/aws/zarf.yaml#L4
https://github.com/defenseunicorns/zarf-package-big-bang/blob/update-repo-name/k3d/zarf.yaml#L4

**Review by mjnagel — APPROVED** [review]
LGTM 🥳

**Pull request merged**